test-requirements.txt

```diff
@@ -1,3 +1,7 @@
 cryptography
 httpretty>=0.9.6
 parameterized>=0.7.0
+coverage==5.0.3
+black==19.10b0; python_version > '3.5'
```

Comment:
I don't think we need the marker here, we require 3.5 or greater anyway

Reply:
This is because black is only available for py3.6+